README.md

```diff
@@ -112,34 +105,38 @@ pip install virtualenv
 ```
 If this does not work, you may need to reboot for the changed PATH settings (by the python installer) to take effect.
 
-#### On Windows mingw
+#### Additional for Windows MSVC
```

Comment:
I think there must be a better way to organize this. Perhaps as a numbered list so that things are grouped and use **(MSVC only)** after the number in order to let them know whether to do that number or skip it?